src/main/java/edu/umich/lib/dor/replicaexperiment/service/ReplicaService.java

```diff
@@ -39,15 +39,19 @@ public Replica getReplica(
             .stream()
             .filter(r -> r.getRepository().getId() == repository.getId())
             .toList();
+        if (matchingReplicas.size() == 0) {
+            return null;
+        }
```

Comment:
Not having this was a bug, calling `getReplica` when no replica existed resulted in an index out of bounds error, I believe.